### PR TITLE
[AutoMM] Fix pdf classficiation tutorial

### PR DESCRIPTION
**Pull request by zhiqiangdon**
*Issue #, if available:*

*Description of changes:*

1. Fixed the missing installation command in the Colab notebook.
2. Removed developer's local environment info from the notebook.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by yinweisu** [comment]
| Previous CI Run | Current CI Run |
| :---: | :---: |
| pydantic_core==2.18.1 | pydantic_core==2.18.2 |
| pydantic==2.7.0 | pydantic==2.7.1 |
| platformdirs==4.2.0 | platformdirs==4.2.1 |
| virtualenv==20.25.3 | virtualenv==20.26.0 |
| pydantic_core==2.18.1 | pydantic_core==2.18.2 |
| pydantic==2.7.0 | pydantic==2.7.1 |
| platformdirs==4.2.0 | platformdirs==4.2.1 |
| virtualenv==20.25.3 | virtualenv==20.26.0 |

File: docs/tutorials/multimodal/document_prediction/pdf_classification.ipynb [hunk 1]
```diff
@@ -230,16 +243,9 @@
...
-   "language": "python",
-   "name": "python3"
-  },
   "language_info": {
-   "name": "python",
-   "version": "3.10.14"
-  },
-  "orig_nbformat": 4
+   "name": "python"
+  }
```
Comment (prateekdesai04) [inline docs/tutorials/multimodal/document_prediction/pdf_classification.ipynb:248]: why did we remove the version specification?

Comment (prateekdesai04) [reply docs/tutorials/multimodal/document_prediction/pdf_classification.ipynb:248]: Would it work on all the versions?

Comment (zhiqiangdon) [reply docs/tutorials/multimodal/document_prediction/pdf_classification.ipynb:248]: Yes, removing the specific version is to be compatible for all versions.

**Comment by yinweisu** [comment]
| Previous CI Run | Current CI Run |
| :---: | :---: |
| botocore==1.34.89 | botocore==1.34.90 |
| boto3==1.34.89 | boto3==1.34.90 |
| botocore==1.34.89 | botocore==1.34.90 |
| boto3==1.34.89 | boto3==1.34.90 |

**Review by prateekdesai04 — APPROVED** [review]
LGTM!

**Comment by yinweisu** [comment]
| Previous CI Run | Current CI Run |
| :---: | :---: |

**Pull request merged**